plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/rename/FbInstanceChange.java

```diff
@@ -0,0 +1,76 @@
+package org.eclipse.fordiac.ide.typemanagement.refactoring.rename;
```

Comment:
Please make sure to add/update the mandatory copyright-header (taking this file as example, applies to all .java-files)